tests/contrib/django/test_django.py

```diff
@@ -1531,6 +1531,46 @@ def test_django_use_handler_resource_format_env(client, test_spans):
         assert out.startswith(b"Test success")
 
 
+@pytest.mark.parametrize("env_val", ["True", "False"])
+def test_django_instrument_databases_env(env_val):
+    """
+    Test that DD_DJANGO_INSTRUMENT_DATABASES enables/disables django db instrumentation
+    """
+    with override_env(dict(DD_DJANGO_INSTRUMENT_DATABASES=env_val)):
+        out = subprocess.check_output(
+            [
+                "ddtrace-run",
+                "python",
+                "-c",
+                (
+                    "from ddtrace import config;"
+                    "assert config.django.instrument_databases is {};print('Test success')".format(env_val)
```

Comment:
you don't need the test success part, as long as the subprocess doesn't exist with an error (which `assert` will  be sure it doesn't), then should be good.

work verifying that `check_output()` will raise an exception on failure though.

Reply:
> assert config.django.instrument_databases is {}

Not sure this makes sense, it should be `True` or `False`

Reply:
I remove the print("test case") and now I just assert if `config.django.instrument_databases` has the same value as the environment variable (true/false). Is that readable or should I split up the test cases?